app/views/issues/_attributes.html.erb

```diff
@@ -743,7 +743,14 @@ function findInput(label, tag = "input") {
   const $label = $(`label:contains(${label})`).filter(function() {
     return $(this).text().trim().indexOf(label) === 0;
   }).first();
-  return $label.length ? $label.siblings(tag).first() : $();
+  if (!$label.length) return $();
+
+  const $siblings = $label.siblings();
+  const $direct = $siblings.filter(tag).first();
+  if ($direct.length) return $direct;
+
+  const $descendant = $siblings.find(tag).first();
+  return $descendant.length ? $descendant : $();
```

Comment:
DOM changes to new Redmine caused checkboxes not to be selected by this method (nested within a `<span>`), so this also checks descendants of siblings. Backward compatible with existing fields because the other inputs are still a direct sibling.